config/script.php

```diff
@@ -3,7 +3,14 @@
 use Clarity\Console\CLI;
 
 return [
-
+    'deploy' => [
```

Comment:
I pushed some fixes to this code, just run `composer update` and this will be interpreted as Closure in the kernel